From 30437a0c8facf5d500a917dc519dedd910e7e656 Mon Sep 17 00:00:00 2001
From: Jordan Speicher <jordan@jspeicher.com>
Date: Tue, 16 Jun 2020 23:28:37 -0500
Subject: [PATCH] Change path

---
 page/script.js | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/page/script.js b/page/script.js
index 802b45c..2064316 100644
--- a/page/script.js
+++ b/page/script.js
@@ -113,9 +113,9 @@ class Server {
     }
 }
 class SnapControl {
-    constructor(host, port) {
+    constructor(path) {
         this.server = new Server();
-        this.connection = new WebSocket('ws://' + host + ':' + port + '/jsonrpc');
+        this.connection = new WebSocket(path);
         this.msg_id = 0;
         this.status_req_id = -1;
         this.connection.onmessage = (msg) => this.onMessage(msg.data);
@@ -255,7 +255,7 @@ class SnapControl {
         }
     }
 }
-let snapcontrol = new SnapControl(window.location.hostname, 1780);
+let snapcontrol = new SnapControl('ws://' + window.location.host + window.location.pathname.split('/').slice(0, -1) + '/jsonrpc');
 //let snapcontrol = new SnapControl("192.168.0.3", 1780);
 let hide_offline = false;
 function show() {
@@ -547,4 +547,3 @@ window.onclick = function (event) {
         modal.style.display = "none";
     }
 };
-//# sourceMappingURL=script.js.map
\ No newline at end of file
-- 
2.25.1

